user/models.py

```diff
@@ -4,15 +4,38 @@
 from django.db import models
 from django.utils import timezone
 
+USR_ORGANIZER = 'O'
+USR_VOLUNTEER = 'V'
+USR_HACKER = 'H'
+USR_MENTOR = 'M'
+USR_SPONSOR = 'S'
+
+USR_TYPE = [
+    (USR_HACKER, 'Hacker'),
+    (USR_MENTOR, 'Mentor'),
+    (USR_SPONSOR, 'Sponsor'),
+    (USR_VOLUNTEER, 'Volunteer'),
+    (USR_ORGANIZER, 'Organizer'),
+]
+
+USR_URL_SPONSOR = 'sponsor'
+USR_URL_TYPE = {
+    'hacker': USR_HACKER,
+    'volunteer': USR_VOLUNTEER,
+    'mentor': USR_MENTOR,
+    USR_URL_SPONSOR: USR_SPONSOR
```

Comment:
Isnt this label.lower()?

Reply:
This is in order to change from 'hacker' to 'H'. I could take the first char and upper() but then if in the future we have more users with 2 chars, this won't work. And I thougth it is more readable like this

Reply:
USR_TYPE has `Hacker` and USR_URL_TYPE has `hacker`. I wonder if that can be collapsed.

Reply:
Done